src/components/IconComponent/index.js

```diff
@@ -0,0 +1,3 @@
+import IconComponent from "./IconComponent";
+
```

Comment:
You can do it like this `export { default } from "./IconComponent";`, it will be shorter

Refactor other _index.js_ files like this

And add rule for _eslintrc.json_ `"no-restricted-exports": "off",`